deployments/rialto/docker-compose.yml

```diff
@@ -215,3 +215,13 @@ services:
       - "4567:4567"
     depends_on:
       - grafana-dashboard
+  front-end:
+    build:
+      context: .
+      dockerfile: ./Front-end.Dockerfile
+      environment:
+        SUBSTRATE_PROVIDER:  ${UI_SUBSTRATE_PROVIDER:-ws://localhost:9933}
```

Comment:
Can you also please add these updated env variables to the docs here: https://github.com/paritytech/parity-bridges-common/blob/27f26a677a7e50b5030fbd7ada8be650ba22d69f/deployments/README.md#environment-variables

Possibly with a comment about what they do.

Reply:
Sure, just added that.